### ISSUE TITLE
Secret Detected in README

### ISSUE THREAD

**Issue by RexBelli** [octokit/auth-token.js#195]
Hello 👋 
In the README on this repo, there is a string that Github Secret Scanning detects as a secret:
```
createTokenAuth("v1.d3d433526f780fbcc3129004e2731b3904ad0b86");
```

I expect this secret is either no longer valid or entirely fake, and so isn't a security concern.  However, it does create an alert whenever someone vendors a version of this repo into their codebase.

![image](https://user-images.githubusercontent.com/20795638/133487487-295f977a-eb3d-4e67-a35f-e36b49478f42.png)

Can this be modified so as to not trigger Github Secret Scanning somehow?  I would propose a PR, but I'm not sure of the exact regex being used to detect Github App Installation Access Tokens.

Thank you!

**Comment by gr2m** [comment]
can you change it to `v1.1234567890abcdef1234567890abcdef12345678`? The secret scanning will run on the pull request so we should find out if it triggers or not

**Comment by RexBelli** [comment]
Sure thing: https://github.com/octokit/auth-token.js/pull/196

**Comment by RexBelli** [comment]
@gr2m It seems it's still detected by Github Secret Scanning:
![Screen Shot 2021-09-15 at 11 45 44 AM](https://user-images.githubusercontent.com/20795638/133491511-77229d7f-1f17-441e-90ab-f28a612844bd.png)

**Comment by gr2m** [comment]
Okay, lets replace all the `1234567890abcdef1234567890abcdef12345678` strings with `secret`. Would you like to send another PR for that?

**Comment by RexBelli** [comment]
Sure, if that's what you want.  If you prefer to keep a real looking secret in the examples, there may be one that doesn't trigger secret scanning.  If you have a contact on the Github Advanced Security team, they may be able to provide some insight.

**Comment by gr2m** [comment]
yes I'll do that, too :) But all the shown tokens have a different format now, so I'll have to update them anyway. I'd iterate on it, lets unblock you first, then I'll update it again once GitHub gets back to me

**Comment by RexBelli** [comment]
Ah, nice!  In that case, I'm fine holding off.  This isn't an pressing issue on our side.

**Comment by gr2m** [comment]
closing in favor of #197 then

**Comment by 15MariamS** [comment]
The new format of GitHub tokens uses a [checksum](https://github.blog/2021-04-05-behind-githubs-new-authentication-token-formats/#checksum); this means that when we scan for GitHub tokens, we can check that the token input matches the checksum and eliminate fake tokens. 

In this case, you can use the checksum to generate an example token that we do not alert on.
